src/main/java/seedu/hirehub/logic/commands/StatusCommand.java

```diff
@@ -28,70 +21,72 @@ public class StatusCommand extends Command {
     public static final String COMMAND_WORD = "status";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates recruitment status of a candidate"
-            + " existing in the address book to one of the five status:\n"
+            + " existing in the list of applications to one of the five status:\n"
             + "PRESCREEN, IN_PROGRESS, WAITLIST, ACCEPTED, REJECTED\n"
-            + "Parameters: INDEX (must be a positive integer) STATUS \n"
-            + "Example: " + COMMAND_WORD + " 2 ACCEPTED";
+            + "Parameters: e/EMAIL ti/jobTitle s/Status \n"
+            + "Example: " + COMMAND_WORD + " e/acekhoon@gmail.com ti/Quantitative Trader s/ACCEPTED";
```

Comment:
The message should be "Update  status for an application within the application list to one of the following statuses"

Reply:
Done